ontology/yaml/resources/LIGHTING/entity_types/LT.yaml

```diff
@@ -45,3 +45,89 @@ LT_DPC:
   implements:
   - LT
   - DPC
+  
+#Entity types for CH ZRH
+LGRP_OS_LC_VADC:
+  id: "4599945020949135360"
```

Comment:
some of these types are duplicated in LGRP.yaml
can you please clean up accordingly?